Podfile

```diff
@@ -23,4 +23,4 @@ end
 
 target 'SnapshotTests' do
   pod 'AccessibilitySnapshot', '0.5.0'
-end
\ No newline at end of file
+end
```

Comment:
Nice

Reply:
That is a part of 71208e1c8e0961100c29d77eb65e78dbabe24100 where I remove linked locally Core SDK source via Cocoapods (committed that by mistake).